PermanentOrg/web-app
# Per 9261 search by tag public archive

This pull request adds the public tag search functionality to the app.

Steps to test:
1. Go to a public archive
2. Type a tag's name in the search bar
3. A dropdown will appear
4. Click on one of the tags
5. The search should be working
